content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -10,187 +10,224 @@ In Odoo, sometimes products included in a single sales order might take stock fr
 click :guilabel:`Create`. From here, the warehouse :guilabel:`Name` and :guilabel:`Short Name` can
 be changed, and other warehouse details can be changed under the :guilabel:`Warehouse Configuration`
 tab.
 
-Under the :guilabel:`Shipments` heading, set the number of steps used to process :guilabel:`Incoming
-Shipments` and :guilabel:`Outgoing Shipments` by selecting between the :guilabel:`1 step`,
-:guilabel:`2 steps`, and :guilabel:`3 steps` radio buttons. The desired option for
-:guilabel:`Incoming Shipments` and :guilabel:`Outgoing Shipments` will depend on the warehouse's
-procurement process, and might differ for individual products or product categories.
+Lastly, click :guilabel:`Save` to finish creating a *regular* warehouse. Continue following the
+steps below to finish configuring the virtual parent warehouse.
+
+.. image:: stock_warehouses/stock-warehouses-create-warehouse.png
+   :align: center
+   :alt: New warehouse form.
 
 .. seealso::
-   - :doc:`How to choose the right flow to handle receipts and deliveries?
-     </applications/inventory_and_mrp/inventory/management/shipments_deliveries>`
-
-Under the :guilabel:`Resupply` heading, configure the method(s) for how the warehouse resupplies
-its inventory:
-
-- :guilabel:`Resupply Subcontractors`: resupply subcontractors with components from this warehouse.
-- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
-  this warehouse.
-- :guilabel:`Manufacture`: to produce right away, move the components to the production location
-  directly and start the manufacturing process; to pick first and then produce, unload the
-  components from the stock to input location first, and then transfer it to the production
-  location.
-- :guilabel:`Buy to Resupply`: when products are bought, they can be delivered to this warehouse.
-- :guilabel:`Resupply From`: automatically create routes to resupply this warehouse from another
-  chosen warehouse
+   - :doc:`Warehouse configurations <../../management/warehouses/warehouses_locations>`
+   - :ref:`Incoming and outgoing shipments <inventory/receipts_delivery_one_step/wh>`
+   - :doc:`Resupply from another warehouse <../../management/warehouses/resupply_warehouses>`
 
-.. tip::
-   *Routes* can be set and configured directly from the :guilabel:`Warehouse` form, by clicking on
-   the :guilabel:`Routes` smart button. Once the warehouse is configured, virtual *Locations* can be
-   created.
+.. _inventory/routes/child-wh:
 
-.. image:: stock_warehouses/stock-warehouses-create-warehouse.png
-   :align: center
-   :alt: The edit screen for creating a new warehouse.
+Create child warehouses
+=======================
 
-In order to apply this virtual warehouse as the *parent* location of two *child* location
-warehouses, there need to be two warehouses configured with physical stock locations.
+Next, create at least two *child* warehouses to link to the virtual warehouse.
```

Comment:
```suggestion
Create at least two *child* warehouses to link to the virtual warehouse.
```
just so these sections can standalone, I'd suggest removing the "Next" from this sentence.